Integrations/ArcSightESM/ArcSightESM.yml

```diff
@@ -319,6 +319,13 @@ script:
     - name: id
       required: true
       description: Resource ID of the query viewer
+    - name: onlyColumns
```

Comment:
remove this arg. this arg was added in as-get-matrixData because of backward compatibility reasons.
We should always return the table with columns in single entry

Reply:
yeah, I added this following a client's request. I can set the default to always return the table with columns in a single entry (onlyColumns=False). would that be ok?